data/json/items/melee/swords_and_blades.json

```
@@ -97,7 +97,22 @@
     "qualities": [ [ "CUT", 2 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 14 ] ],
     "techniques": [ "RAPID" ],
     "flags": [ "STAB" ],
-    "weapon_category": [ "KNIVES" ]
+    "weapon_category": [ "KNIVES" ],
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "switchblade_balisong",
+        "name": { "str": "balisong" },
+        "description": "A flashy knife with hinged handles that completely conceal the blade while folded. Popular among movie gangbangers, and people who thought fidget spinners were too safe.",
```

The typography convention for in-game text is to put two spaces between sentences: https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md
```suggestion
        "description": "A flashy knife with hinged handles that completely conceal the blade while folded.  Popular among movie gangbangers, and people who thought fidget spinners were too safe.",
```